README.md

```diff
@@ -46,6 +48,22 @@ bal graphql [-i | --input] <graphql-service-file-path> [-o | --output] <output-l
 
+### Command for Ballerina GraphQL service generation
+
+The `graphql` command for Ballerina Graphql service generation and the usages are as follows.
+
+```
+bal graphql [-i | --input] <graphql-schema-file-path> [-o | --ouput] <output-location> [-m | --mode] <operation-mode> [-r | --use-records-for-objects]
+```
+
+
+| Argument                      | Description                                                                                                                                                                                                                                                                                                                                                                    |
+|-------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| -i, --input                   | The `input` parameter specifies the path of the GraphQL schema file (e.g., schema.graphql). This parameter is mandatory.                                                                                                                                                                                                                                                       |
+| -o, --output                  | The `output` parameter specifies the path of the output location of the generated Ballerina GraphQL service files. This parameter is optional. If this parameter is not specified, the service files will be generated at the same location from which the GraphQL command is executed.                                                                                        |
+| -m, --mode                    | The `mode` parameter specifies the operation mode. It can be `client`, `schema`, or `service`. They represent Ballerina GraphQL client generation, GraphQL schema generation, and Ballerina GraphQL service generation respectively. The `mode` flag is optional. If the `mode` flag is not specified, the `graphql` tool will infer the mode from the `input` file extension. |
+| -r, --use-records-for-objects | The `use-records-for-objects` flag makes the GraphQL tool use record types for GraphQL object types whenever possible in Ballerina GraphQL service generation. This flag is optional.                                                                                                                                                                                          |
```

Comment:
```suggestion
| -r, --use-records-for-objects | The `use-records-for-objects` flag makes the GraphQL tool use record types for GraphQL object types whenever possible in Ballerina GraphQL service generation. This flag is optional and it can only be used with the GraphQL service generation.                                                                                                                                                                                          |
```